tests/integration.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::{Context, Result};
+use nix::{
+    sys::signal::{kill, SIGINT},
+    unistd::Pid,
+};
+use std::{fs, process::Command, thread, time::Duration};
+
+// The maximum amount of allowed VmRSS in Kilobytes
+const MAX_RSS_KB: u32 = 4000;
+
+// We assume that the tests run in release mode
+const SERVER_BINARY: &str = "target/release/conmon-server";
+const CLIENT_BINARY: &str = "target/release/conmon-client";
+
+#[test]
```

Comment:
if we migrate to #[tokio::test] we can leverage more of tokio by using tokio::process::Command.

Reply:
I’ll change it.